app/src/test/java/org/mozilla/focus/utils/UriExtensionsTest.kt

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils
+
+import android.net.Uri
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class UriExtensionsTest {
+    @Test
+    fun testTruncatedHost() {
+        assertTruncatedHost("mozilla.org", "https://www.mozilla.org")
```

Comment:
Couldn't I do that in Java too? I'm not 100% convinced that we gain much here. We'll have a method less. But we also made the other method a little bit more complex.

Reply:
In retrospect, I can see that neither is clearly better. I like the map literal because it's marginally easy to scan what is being tested and to add new things to test (e.g. you can type it out instead of having to copy-pasta & edit).

Java doesn't have map literals so it's more verbose:
```java
final Map<String, String> expectedToUrl = new HashMap<>();
expectedToUrl.add("mozilla.org", "https:www.mozilla.org");
...
```